jicoco/src/main/java/org/jitsi/xmpp/mucclient/MucClient.java

```diff
@@ -295,7 +295,12 @@ public void connectionClosedOnError(Exception e)
                 logger.warn("Closed on error:", e);
             }
 
-            @Override
+            /**
+             * The connection has reconnected successfully to the server. Connections will
+             * reconnect to the server when the previous socket connection was abruptly closed.
+             * @deprecated use {@link #connected(XMPPConnection)} or {@link #authenticated(XMPPConnection, boolean)} instead.
+             */
+            @Deprecated
```

Comment:
This method has been removed from smack. Since it contains some functionality I've only marked it as deprecated.
Was that correct? Or should we keep the method regardless of what smack did?